pkg/disk/filesystem.go

```diff
@@ -7,6 +7,10 @@ import (
 	"github.com/google/uuid"
 )
 
+// filesystem implements expected interface
+var _ = Mountable(&Filesystem{})
+var _ = UniqueEntity(&Filesystem{})
```

Comment:
The other checks were moved to tests but these weren't.  Is that accidental?

Reply:
Thanks, silly me - that was an oversight. Will fix :)